src/js/mobile-template.js

```diff
@@ -0,0 +1,88 @@
+var mobileTL =
+
+`
+<div class="header">
+    <img src="${logo}" />
+        <a href="#">Twitter</a>
+        <a href="#">Facebook</a>
+        <a href="#">Instagram</a>
+</div>
+<div class="mobile-body">
+    <div class="news">
+
+    </div>
+    <div class="specials">
+
+    </div>
+    <div class="location">
+
+    </div>
+    <div class="story">
+
+    </div>
+    <div class="menu">
+        <li class="entrees">
```

Comment:
-you put a li in a div.

-we need to forEach over the sets of data to generate all the items in the array, not just one:

data.beer.forEach(function(beer){
   $('.container').append('
       // template here
  ';
})

-then we need to do that for entrees and games as well.